self-hosted/Dockerfile

```diff
@@ -74,6 +74,7 @@ RUN set -x \
   && mv dogstatsd_plugin.so /var/lib/uwsgi/ \
   && rm -rf /tmp/requirements-frozen.txt /tmp/uwsgi-dogstatsd .uwsgi_plugins_builder \
   && apt-get purge -y --auto-remove $buildDeps \
+  && apt-get install -y htop gcc && pip install madbg \
```

Comment:
I assume this is temporary for debugging and will be removed before merging?

Reply:
mea culpa